docs/programming/tilebuffer.md

```diff
@@ -0,0 +1,285 @@
+# Multi-directional Scrolling (TileBuffer)
+
+## Basic description
+
+This tutoriel only focus on multi-directional scolling as used for RPG or games like Alien Breed, Chaos Engine or large platform game like Turrican or Superfrog.
+
+In order to do this multi-directional scolling, in ACE you need to use  the [managers/viewport/tilebuffer.h](../../include/ace/managers/viewport/tilebuffer.h). It manages for tiled background/map as a main entry.
```

Comment:
```suggestion
Although you could use simpleBuffer for drawing and scrolling large tilemaps, it would eat a lot of RAM. In order to do this efficiently, you need to use the [managers/viewport/tilebuffer.h](../../include/ace/managers/viewport/tilebuffer.h). It's specifically design for drawing tiles and scrolling in any direction.
```